modules/500-upmeter/openapi/doc-ru-config-values.yaml

```
@@ -125,7 +129,7 @@ properties:
             properties:
               secretName:
                 description: |
-                  Имя Secret'а в namespace `d8-system`, который будет использоваться для smoke-mini.
+                  Имя Secret в namespace `d8-system`, который будет использоваться для smoke-mini.
```

```suggestion
                  Имя Secret в namespace `d8-system`, который будет использоваться для smoke-mini.
```
@elenashliaga @MariaMakeeva У нас в редполитике же уже допускается использование _секрет_?

Тогда можно сделать например: Имя секрета (Secret) в пространстве имен `d8-system`, который будет использоваться для smoke-mini.
или типа того

---

Да, можно.  Или оставить русскоязычный - "секрет". Если меняем namespace на "пространство имен" - потом линтером нужно везде будет пофиксить. В общем, после общения с тим-индией зафиксируем новый глоссарий, который предлагаю расширять в двльнейшем в тим-док))